src/libraries/System.IO.FileSystem/tests/Enumeration/SkipAttributeTests.cs

```diff
@@ -25,21 +25,24 @@ public void SkippingHiddenFiles()
         {
             DirectoryInfo testDirectory = Directory.CreateDirectory(GetTestFilePath());
             DirectoryInfo testSubdirectory = Directory.CreateDirectory(Path.Combine(testDirectory.FullName, GetTestFileName()));
-            FileInfo fileOne = new FileInfo(Path.Combine(testDirectory.FullName, GetTestFileName()));
 
-            // Put a period in front to make it hidden on Unix
-            FileInfo fileTwo = new FileInfo(Path.Combine(testDirectory.FullName, "." + GetTestFileName()));
+            FileInfo fileOne = new FileInfo(Path.Combine(testDirectory.FullName, GetTestFileName()));
```

Comment:
No changes in this test, just organized some lines for easier understanding.
It also helps to show that this test is affected by my PR.